content/challenges/easy/10 - Add up the Numbers/func.js

```diff
@@ -8,5 +8,10 @@
  * @returns {number} sum of all numbers from 1 to num
  */
 module.exports = (num) => {
-
+    let total = 0
+    for (i = 1; i <= num; i++) {
+        total += i
+    }
+    return total
```

Comment:
Thats one way to solve it! Now try learning .reduce()

https://www.w3schools.com/jsref/jsref_reduce.asp